ci/build_and_push_docker_image.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+export otp_vsn="${OTP_RELEASE:-25.3}"
+echo "ERLANG/OTP ${otp_vsn}"
+
+docker buildx build --platform linux/amd64,linux/arm64 \
+       --build-arg otp_vsn \
+       --push -t mongooseim/amoc-arsenal-xmpp:latest .
```

Comment:
- I kept the standard builds for testing because they are much faster, and we know quicker whether it works or not (10 minutes of difference on CI). The `buildx` command would need to be modified to load the image instead of pushing it.
- Regarding tags, I just didn't think we need them at this point...